<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TilesetEditor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(65, 81, 92);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar { /* Default style with no interaction with the user */
	background-color: rgb(52, 66, 74);
    spacing: 15px; /* spacing between menu bar items */
}

QMenuBar::item {
    background: transparent;
}

QMenuBar::item:selected { /* when mouse or keyboard is hovering */
	background: #41515c;
}

QMenuBar::item:pressed { /* When the menu is pressed and open */
    background: rgb(86, 145, 236);
}


/* Menu items */

QMenu {
    background-color: #34424a; /* sets background of the menu */
}

QMenu::item {
    /* sets background of menu item. set this to something non-transparent
        if you want menu color and menu item color to be different */
    background: transparent;
}

QMenu::item:selected { /* when user selects item using mouse or keyboard */
    background: #41515c;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuExecute">
    <property name="title">
     <string>Execute</string>
    </property>
    <addaction name="actionReload_dump"/>
    <addaction name="actionBuild_tilesets"/>
    <addaction name="actionPipelines"/>
    <addaction name="actionEncode_HD_tiles"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExecute"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open Context</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Context</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionReload_dump">
   <property name="text">
    <string>Reload dump</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionBuild_tilesets">
   <property name="text">
    <string>Build tilesets</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionPipelines">
   <property name="text">
    <string>Pipelines</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionEncode_HD_tiles">
   <property name="text">
    <string>Encode HD tiles</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
